apps/charterafrica/src/payload/blocks/EmbeddedChart.js

```diff
@@ -0,0 +1,92 @@
+import { validateHexColor } from "../utils/colors";
+
+const EmbeddedChart = {
+  slug: "embedded-chart",
+  labels: {
+    singular: {
+      en: "Flourish Charts",
+      fr: "Graphiques Flourish",
+      pt: "Gráficos Flourish",
+    },
+    plural: {
+      en: "Flourish Charts",
+      fr: "Graphiques Flourish",
+      pt: "Gráficos Flourish",
+    },
+  },
+  fields: [
+    {
+      name: "title",
+      label: {
+        en: "Title",
+        fr: "Titre",
+        pt: "Título",
+      },
+      type: "text",
+      required: true,
+      localized: true,
+    },
+    {
+      name: "subtitle",
+      label: {
+        en: "Subtitle",
+        fr: "Sous-titre",
+        pt: "Subtítulo",
+      },
+      type: "text",
+      required: true,
+      localized: true,
+    },
+    {
+      name: "html",
+      label: {
+        en: "Embed Code",
+        fr: "Code d'intégration",
+        pt: "Código de incorporação",
+      },
+      type: "textarea",
```

Comment:
Why did you just ignore https://github.com/CodeForAfrica/ui/pull/469#discussion_r1244806021?

Reply:
😮